src/server/main.js

```diff
@@ -1,3 +1,26 @@
-import express from './express';
+import config from './config';
+import express from 'express';
+import {Server} from 'http';
 
-express();
+const app = exports.app = express();
+const server = exports.server = Server(app);
+
+// TODO: Add API sub-app to demonstrate other use cases
+// app.use('/api/v1', require('./api'));
+
+// Load react-js frontend
+app.use(require('./frontend'));
+
+// Add error handler
+// Four arguments need to be defined in order for the middleware to act as an error handler
+app.use((err, req, res, next) => {
+  const msg = err.stack || err;
+  console.log('Yay', msg);
+  res.status(500).send('500: ' + msg);
+});
```

Comment:
Global error handler but route/middleware specific ones can still be created and in case they fail to handle the error, they can always use `next(err)` to bubble it up.